about.css

```diff
@@ -0,0 +1,439 @@
+  top: 0;
+  height: 90vh;
+  width: 100%;
+  z-index: 2;
+  background: rgba(85, 85, 87, 0.8);
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+}
+
+.menu-ul {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.menu-ul li {
+  list-style: none;
+  margin: 10px auto;
+}
+
+.menu-ul li a {
+  color: #fff;
+  font-size: 1.5rem;
+  text-decoration: none;
+  transition: all 0.35s ease-in-out;
+}
+
+.menu-ul li a:hover {
+  color: #ec5242;
+  font-size: 2rem;
+}
+
+.cancel-icon {
+  position: absolute;
+  top: 0;
+  left: 90%;
+  color: #fff;
+}
+
+button {
+  border: none;
+}
+
+main {
+  padding: 1.5rem;
+}
+
+.section-1 {
+  margin: 20px auto;
+  padding: 1rem;
+  max-width: 90%;
+}
+
+.section-1 > h1,
+.section-1 h2 {
+  color: #ec5242;
+  text-align: center;
+}
+
+.section-1 h1 {
+  font-size: 2rem;
+  margin: 0.5rem;
+}
+
+.section-1 > p {
+  background: #f5f4f4;
+  border: #d3d3d3 solid 1.5px;
+  margin: 2rem auto;
+  padding: 0.5rem;
+  text-align: center;
+  font-weight: bold;
+}
+
+.section-1 p {
+  text-align: center;
+}
+
+.new {
+  display: block;
+  text-align: center;
+  text-decoration: underline;
+  text-decoration-color: #d3d3d3;
+}
+
+.bold {
+  font-weight: 800;
+}
+
+.section-2 {
+  width: 80%;
+  margin: 2rem auto;
+  margin-top: 4rem;
+  padding: 0.5rem;
+}
+
+.section-2 h2 {
+  text-align: center;
+  width: 80%;
+  margin: auto;
+}
+
+.section-2 > p {
+  text-align: center;
+  margin: 0.5rem;
+}
+
+.section-3 {
+  width: 90%;
+  border: #d3d3d3 1px solid;
+  margin: 4rem auto;
+  margin-top: 2rem;
+  display: flex;
+  justify-content: center;
+  padding: 1.5rem;
+}
+
+.section-3 i {
+  align-self: center;
+}
+
+.section-3 div {
+  width: 50%;
+  margin: 0 0.5rem;
+}
+
+.section-3 p {
+  text-decoration: underline overline #d3d3d3 3px;
+  margin: 0.3rem auto;
+  width: 100%;
+}
+
+.section-4 {
+  width: 80%;
+  margin: 2rem auto;
+}
+
+.section-4 > h2,
+.section-4 > p {
+  margin: 1rem auto;
+  text-align: center;
+}
+
+.past-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.past {
+  height: 300px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 20px auto;
+}
+
+.asia {
+  background: linear-gradient(to bottom, #ec534293, #ec534294), url(./images/Asia.jpg) center center/cover;
+}
+
+.asia h2,
+.asia p {
+  color: #fff;
+  margin: 10px auto;
+}
+
+.europe {
+  background: linear-gradient(to bottom, #ec534293, #ec534294), url(./images/europe.jpg) center center/cover;
+}
+
+.europe h2,
+.europe p {
+  color: #fff;
+  margin: 10px auto;
+}
+
+/* .past-event img:hover {
+  opacity: 1;
+}
+
+.past-event .past-event-text {
+  width: 80%;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: #272a31;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+} */
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments.